src/main/java/Duke/TaskList.java

```diff
@@ -0,0 +1,9 @@
+package Duke;
+
+import Duke.Tasks.Task;
+
+import java.util.ArrayList;
+
+public class TaskList extends ArrayList<Task>{
+
+}
```

Comment:
Perharps TaskList can be used to abstract out the insertion, deletion and manipulation of the tasks?

Reply:
Can you elaborate? Because I feel that insertion and deletion etc can already be handled by ArrayList.